### PR TITLE
mgr: prometheus: added osd commit/apply latency metrics (#22718).

### PR DESCRIPTION
**Pull request by k0ste**
Prometheus module lack of this metrics.

http://tracker.ceph.com/issues/22718

### CODE REVIEW

File: src/pybind/mgr/prometheus/module.py [hunk 1]
```diff
@@ -202,6 +204,15 @@ def _setup_static_metrics(self):
                 'OSD status {}'.format(state),
                 ('ceph_daemon',)
             )
+        for state in OSD_STATS:
```
Comment (jcsp) [inline src/pybind/mgr/prometheus/module.py]: s/state/stat/

**Comment by jan--f** [comment]
Other then @jcsp's comment (also applies [here](https://github.com/ceph/ceph/pull/19980/files#diff-aa300f60a885cd98ccce56607850eef0R294)) LGTM

**Comment by k0ste** [comment]
ready_for_review

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2018-01-22_16:28:42-rados-wip-sage3-testing-2018-01-22-0845-distro-basic-smithi/#

**Pull request merged**